boost-common/src/test/java/io/openliberty/boost/common/config/BoosterConfiguratorTest.java

```diff
@@ -8,78 +8,82 @@
  * Contributors:
  *     IBM Corporation - initial API and implementation
  *******************************************************************************/
-package io.openliberty.boost.common.utils;
+package io.openliberty.boost.common.config;
 
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertTrue;
 
-import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 import java.util.Properties;
 
+import org.junit.Rule;
 import org.junit.Test;
-import org.mockito.Mockito;
+import org.junit.contrib.java.lang.system.RestoreSystemProperties;
 
+import io.openliberty.boost.common.BoostException;
 import io.openliberty.boost.common.BoostLoggerI;
+import io.openliberty.boost.common.boosters.AbstractBoosterConfig;
+import io.openliberty.boost.common.boosters.JDBCBoosterConfig;
 import io.openliberty.boost.common.config.BoostProperties;
-import io.openliberty.boost.common.config.BoosterPackConfigurator;
-import io.openliberty.boost.common.config.JDBCBoosterPackConfigurator;
-
-public class LibertyBoosterUtilTest {
+import io.openliberty.boost.common.config.BoosterConfigurator;
+import io.openliberty.boost.common.utils.BoosterUtil;
+import io.openliberty.boost.common.utils.CommonLogger;
+
+public class BoosterConfiguratorTest {
+    
+    @Rule
+    public final RestoreSystemProperties restoreSystemProperties = new RestoreSystemProperties();
+    
```

Comment:
What's this  RestoreSystemProperties  for?   Can you maybe add a comment explaining?

Reply:
Possibly could be post-merge...(next PR).

Reply:
This is documented in the PR description, but sure will add a comment here as well.